packages/ag-charts-community/src/options/chart/chartBuilderOptions.ts

```diff
@@ -0,0 +1,31 @@
+import type { AgBaseCartesianChartOptions } from '../series/cartesian/cartesianOptions';
+import type { AgBaseHierarchyChartOptions } from '../series/hierarchy/hierarchyOptions';
+import type { AgBasePolarChartOptions } from '../series/polar/polarOptions';
+import type { AgBaseChartOptions } from './chartOptions';
+import type { AgBaseChartThemeOptions, AgChartTheme, AgChartThemeName } from './themeOptions';
+
+export interface AgChartThemeOptions extends AgBaseChartThemeOptions {}
+export type AgChartThemeOverrides = NonNullable<AgChartThemeOptions['overrides']>;
+
+export interface AgCartesianChartOptions extends AgBaseCartesianChartOptions, AgBaseChartOptions {
+    theme?: AgChartTheme | AgChartThemeName;
+}
+export interface AgPolarChartOptions extends AgBasePolarChartOptions, AgBaseChartOptions {
+    theme?: AgChartTheme | AgChartThemeName;
+}
+export interface AgHierarchyChartOptions extends AgBaseHierarchyChartOptions, AgBaseChartOptions<any> {
+    theme?: AgChartTheme | AgChartThemeName;
+}
+export type AgChartOptions = AgCartesianChartOptions | AgPolarChartOptions | AgHierarchyChartOptions;
+
+export interface AgChartInstance {
+    className?: string;
```

Comment:
Is this new? It probably shouldn't be in the public contract as our classnames are part of our internal implementation.